packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -66,6 +68,10 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [],
   );
 
+  const [virtualCursorElements, setVirtualCursorElements] = React.useState([]);
```

Comment:
nit: please use a type here (might not be correct in the suggestion)
```suggestion
  const [virtualCursorElements, setVirtualCursorElements] = React.useState<HTMLElement>([]);
```

Reply:
Okay, fixed.